### ISSUE TITLE
AttributeError: 'FlakesChecker' object has no attribute 'JOINEDSTR'

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#172]
In GitLab by @dimaqq on Oct 27, 2016, 06:41

*Please provide the following version information*

- Version of Python which has Flake8 installed (e.g., `python -V`)
 - Python 3.6.0b2
- How you installed Flake8 (e.g., `pip`, `apt`, `yum`, etc.)
 - pip install pytest-flake8
- If you used pip, the version of pip installed (e.g., `pip --version`)
 - pip 8.1.2 from /dima/.pyenv/versions/3.6.0b2/envs/turbo/lib/python3.6/site-packages (python 3.6)
- Version of setuptools installed (e.g., `python -c 'import setuptools; print(setuptools.__version__)'`)
 - 27.1.2
- Version of Flake8 installed (i.e., copy and paste all of the output from `flake8 --version`)
 - 3.0.4 (mccabe: 0.5.2, pyflakes: 1.2.3, pycodestyle: 2.0.0) CPython 3.6.0b2 on Linux

```
$▶pip freeze | grep flake
flake8==3.0.4
pyflakes==1.2.3
pytest-flake8==0.8.1
```

same with

```
$▶pip freeze | grep flake
flake8==3.0.4
pyflakes==1.3.0
pytest-flake8==0.8.1
```

*Please describe the problem or feature*

Following input cannot be parsed: `f'={version}'`

```
Process Process-1:
Traceback (most recent call last):
  File "/dima/.pyenv/versions/3.6.0b2/lib/python3.6/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/dima/.pyenv/versions/3.6.0b2/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/dima/.pyenv/versions/3.6.0b2/envs/turbo/lib/python3.6/site-packages/flake8/checker.py", line 633, in _run_checks_from_queue
    checker.run_checks(results_queue, statistics_queue)
  File "/dima/.pyenv/versions/3.6.0b2/envs/turbo/lib/python3.6/site-packages/flake8/checker.py", line 573, in run_checks
    self.run_ast_checks()
  File "/dima/.pyenv/versions/3.6.0b2/envs/turbo/lib/python3.6/site-packages/flake8/checker.py", line 477, in run_ast_checks
    checker = self.run_check(plugin, tree=ast)
  File "/dima/.pyenv/versions/3.6.0b2/envs/turbo/lib/python3.6/site-packages/flake8/checker.py", line 457, in run_check
    return plugin['plugin'](**arguments)
  File "/dima/.pyenv/versions/3.6.0b2/envs/turbo/lib/python3.6/site-packages/flake8/plugins/pyflakes.py", line 72, in __init__
    withDoctest=with_doctest)
  File "/dima/.pyenv/versions/3.6.0b2/envs/turbo/lib/python3.6/site-packages/pyflakes/checker.py", line 426, in __init__
    self.runDeferred(self._deferredFunctions)
  File "/dima/.pyenv/versions/3.6.0b2/envs/turbo/lib/python3.6/site-packages/pyflakes/checker.py", line 463, in runDeferred
    handler()
  File "/dima/.pyenv/versions/3.6.0b2/envs/turbo/lib/python3.6/site-packages/pyflakes/checker.py", line 1055, in runFunction
    self.handleNode(stmt, node)
  File "/dima/.pyenv/versions/3.6.0b2/envs/turbo/lib/python3.6/site-packages/pyflakes/checker.py", line 794, in handleNode
    handler(node)
  File "/dima/.pyenv/versions/3.6.0b2/envs/turbo/lib/python3.6/site-packages/pyflakes/checker.py", line 747, in handleChildren
    self.handleNode(node, tree)
  File "/dima/.pyenv/versions/3.6.0b2/envs/turbo/lib/python3.6/site-packages/pyflakes/checker.py", line 793, in handleNode
    handler = self.getNodeHandler(node.__class__)
  File "/dima/.pyenv/versions/3.6.0b2/envs/turbo/lib/python3.6/site-packages/pyflakes/checker.py", line 627, in getNodeHandler
    self._nodeHandlers[node_class] = handler = getattr(self, nodeType)
AttributeError: 'FlakesChecker' object has no attribute 'JOINEDSTR'
```

For comparison, `f'{version}'`gets parsed correctly. Why is equals sign special in `={` ?

*If this is a bug report, please explain with examples (and example code) what you expected to happen and what actually happened.*

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Oct 27, 2016, 07:19

This is a problem you should report upstream to [pyflakes](https://github.com/pycqa/pyflakes).

**Comment by asottile** [comment]
In GitLab by @dimaqq on Oct 27, 2016, 07:41

Looks like https://bugs.launchpad.net/pyflakes/+bug/1624098 already exists.

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Oct 27, 2016, 07:54

So we're waiting on a couple releases then.

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Nov 12, 2016, 11:46

pyflakes 1.3.0 was released and the cap has been raised for 3.1.0. The "fix" for this has been committed to master.
